packages/svelte-ux/src/lib/utils/date.test.ts

```diff
@@ -30,21 +33,21 @@ const fr = createLocaleSettings({
     dates: {
       ordinalSuffixes: {
         one: 'er',
-        two: '',
-        few: '',
-        other: '',
       },
     },
   },
 });
 
 describe('formatDate()', () => {
   it('should return empty string for null or undefined date', () => {
+    // @ts-expect-error
```

Comment:
Should this error?

Reply:
Yes, because the number of args doesn't match anymore.